book/src/raw-block.md

```diff
@@ -11,22 +11,62 @@ A *mounted (file) volume* is volume that will be mounted using a specified file
 
 The [CSI spec](https://github.com/container-storage-interface/spec/blob/master/spec.md) supports both block and mounted (file) volumes.
 
-While Kubernetes support of mounted (file) volumes is GA/stable, support for [block volume in Kubernetes](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#raw-block-volume-support) was introduced in v1.9, and promoted to beta in Kubernetes 1.13.
+While Kubernetes support of mounted (file) volumes is GA/stable, support for [block volume in Kubernetes](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#raw-block-volume-support) was introduced in v1.9, and promoted to beta in Kubernetes v1.13. The CSI-specific portions of block volumes was introduced in v1.11 promoted to beta in v1.14.
```

Comment:
To avoid confusion, I think we should leave out the in-tree support versions and only talk about versions appropriate to CSI.